docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md

```diff
@@ -0,0 +1,563 @@
+      storage: 1Gi
+  storageClassName: openebs-single-replica
+EOF
+persistentvolumeclaim/ms-volume-claim created
+```
+
+As a next step verify the PV/PVC and the Replicated PV Mayastor volumes.
+
+**Command**
+
+```
+oc get pvc
+```
+
+**Output**
+
+```
+NAME              STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS             VOLUMEATTRIBUTESCLASS   AGE
+ms-volume-claim   Bound    pvc-144d54db-a3cf-4194-821d-34eae9dafc1d   1Gi        RWO            openebs-single-replica   <unset>                 40s
+```
+
+**Command**
+
+```
+oc get pv
+```
+
+**Output**
+
+```
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                            STORAGECLASS             VOLUMEATTRIBUTESCLASS   REASON   AGE
+pvc-02333bf8-8a07-4ce0-a00e-bd6bc67af380   2Gi        RWO            Delete           Bound    openebs/data-openebs-etcd-0      mayastor-etcd-localpv    <unset>                          47h
+pvc-144d54db-a3cf-4194-821d-34eae9dafc1d   1Gi        RWO            Delete           Bound    default/ms-volume-claim          openebs-single-replica   <unset>                          42s
+pvc-233aafb1-59e9-4836-b8a1-f74ab2f5a6e4   10Gi       RWO            Delete           Bound    openebs/storage-openebs-loki-0   mayastor-loki-localpv    <unset>                          47h
+```
+
+**Command**
+
+```
+kubectl mayastor get volumes -n openebs
+```
+
+**Output**
+
+```
+ID                                    REPLICAS  TARGET-NODE  ACCESSIBILITY  STATUS  SIZE  THIN-PROVISIONED  ALLOCATED  SNAPSHOTS  SOURCE
+144d54db-a3cf-4194-821d-34eae9dafc1d  1         <none>       <none>         Online  1GiB  false             1GiB       0          <none>
+```
+
+The Replicated PV Mayastor CSI driver will cause the application pod and the corresponding Replicated PV Mayastor volume's NVMe target/controller ("Nexus") to be scheduled on the same Replicated PV Mayastor Node, to assist with the restoration of volume and application availability in the event of a storage node failure.
+
+```
+kind: Pod
+apiVersion: v1
+metadata:
+  name: fio
+spec:
+  nodeSelector:
+    openebs.io/engine: mayastor
+  volumes:
+    - name: ms-volume
+      persistentVolumeClaim:
+        claimName: ms-volume-claim
+  containers:
+    - name: fio
+      image: nixery.dev/shell/fio
+      args:
+        - sleep
+        - "1000000"
+      volumeMounts:
+        - mountPath: "/volume"
+          name: ms-volume
+```
+
+Verify the application.
+
+**Command**
+
+```
+oc get pods fio
+```
+
+**Output**
+
+```
+NAME   READY   STATUS    RESTARTS   AGE
+fio    1/1     Running   0          34s
+```
+
+Execute the Fio-tester application: The FIO Test utility is now run against the Replicated PV Mayastor for 60 seconds to verify that I/O operations are handled as expected and without errors. In this example, the test uses a pattern of random reads and writes, with a block size of 4KB, and a queue depth of 16.
```

Comment:
Do we need to show this here or should we leave it at the pod is running?
This feels quite low-level output to put in a user guide, not sure if it adds any value for a user CC @avishnu

Reply:
Hmm ya, no need for this.

Reply:
@balaharish7 I think @tiagolobocastro is suggesting not to shw the entire output.

Reply:
I've removed the entire output from "Run Fio-tester application"